### ISSUE TITLE
Preview resolution of covers and show highest resolution in another color

### ISSUE THREAD

**Issue by mr44er** [kellnerd/harmony#38]
Preview resolution of covers and show highest resolution in another color (green?) to pick fast.
I know this generates more traffic, but I open them manually in tabs anyway to compare.

**Comment by UltimateRiff** [comment]
I don't know if simply highest resolution is always best, as I know some stores do upscale images using various means, or some might use lossy compression

that said, definitely second showing resolution and filetype (and perhaps whether the filetype uses lossy or lossless compression)

**Comment by mr44er** [comment]
Yeah, resolution only isn't always the best decision, but helps where to look first...instead of opening 3-4 tabs and not knowing what to expect first.

Another idea would be the presentation order after the check. Sort order only for resolution, but additionally filesize and format png/jpg shown. From left/highres to the right/lowres.
Quick example I have in mind:

${\textsf{\color{green}4000x4000px +}}$ / 7,1MiB / .png >> ${\textsf{\color{orange}1000x1000px}}$ / 0,8MiB / .jpg >> ${\textsf{\color{red}600x600px}}$ / 0,5MiB / .jpg

**Comment by mr44er** [comment]
Checking only the extension isn't sufficient, it seems. There are rare cases (so far only noticed on bandcamp) where mime-type is wrong, so the check should do this...again more traffic and latency :)

https://adabroc.bandcamp.com/album/iolaire -> https://f4.bcbits.com/img/a1217632439_0.jpg

```
mr44er@ws01:/usr/home/mr44er/cdcover $ file a1217632439_0.jpg 
a1217632439_0.jpg: PNG image data, 1210 x 1210, 8-bit/color RGBA, non-interlaced
```

**Comment by kellnerd** [comment]
I think this has been addressed by https://github.com/ROpdebee/mb-userscripts/pull/811

If you still want highlighting of the "best" cover image, this would be a feature request for the userscript which benefits both the Harmony and the a-tisket integration.
